packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -523,6 +530,7 @@ export default async function main(
       deployStamp,
       target,
       skipAutoDetectionConfirmation: autoConfirm,
+      projectSettings: { sourceFilesOutsideRootDirectory },
```

Comment:
We need this change in `vc link` too https://github.com/vercel/vercel/blob/0fc74feddcf0a5777ead14738171ad2dfd2e7077/packages/now-cli/src/util/link/setup-and-link.ts#L177-L186